woocommerce-gateway-stripe.php

```diff
@@ -54,6 +54,73 @@ function woocommerce_stripe_wc_not_supported() {
 	echo '<div class="error"><p><strong>' . sprintf( esc_html__( 'Stripe requires WooCommerce %1$s or greater to be installed and active. WooCommerce %2$s is no longer supported.', 'woocommerce-gateway-stripe' ), WC_STRIPE_MIN_WC_VER, WC_VERSION ) . '</strong></p></div>';
 }
 
+/**
+ * WooCommerce country not supported notice.
+ *
+ * @since 4.5.1
+ * @return string
+ */
+function woocommerce_stripe_wc_country_not_supported() {
+	echo '<div class="error"><p><strong>' . __( 'Stripe is not supported in your store\'s country.', 'woocommerce-gateway-stripe' ) . '</strong></p></div>';
```

Comment:
> What do you think about creating a file for the tests setup and requiring it after loading the plugin files?
Yeah - that's a good idea. It will make it easier to maintain that way. Will do.

Reply:
> It's clear to me from the message that the plugin won't also work because of the store's country, but should we mention that it won't be available because of that?

I'll mention that too. Good idea. Thanks!

Reply:
Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1254/commits/dcf9ff3d37284dd381f7878c071c6d8b5e35986c